lib/beaker-hostgenerator/data.rb

```diff
@@ -1,11 +1,21 @@
 module BeakerHostGenerator
   module Data
 
-    # Pull various informations out of the environment.
-    PE_VERSION=ENV['pe_version']
-    PE_FAMILY=ENV['pe_family']
-    PE_UPGRADE_VERSION=ENV['pe_upgrade_version']
-    PE_UPGRADE_FAMILY=ENV['pe_upgrade_family']
+    def pe_version
+      ENV['pe_version']
+    end
+
+    def pe_family
+      ENV['pe_family']
+    end
+
+    def pe_upgrade_version
+      ENV['pe_upgrade_version']
+    end
+
+    def pe_upgrade_family
+      ENV['pe_upgrade_family']
+    end
```

Comment:
Historicaly reasons mostly:
https://github.com/puppetlabs/sqa-utils-gem/blob/master/bin/genconfig#L18-L22

I'm definitely willing to consider changing their names for beaker-hostgenerator 1.x but doing so will have implications throughout CI and probably user scripts.